docs/references/backend/organization/create-organization-invitation-bulk.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: '`createOrganizationInvitationBulk()`'
+description: Use Clerk's Backend SDK to create multiple invitations for new users to join an organization.
+---
+
+{/* clerk/javascript file: https://github.com/clerk/javascript/blob/main/packages/backend/src/api/endpoints/OrganizationApi.ts#L292-L303 */}
+
+Creates multiple [`OrganizationInvitation`](/docs/references/backend/types/backend-organization-invitation)s in bulk for new users to join an organization.
+
+```ts
+function createOrganizationInvitationBulk(
+  organizationId: string,
+  params: CreateBulkOrganizationInvitationParams,
+): Promise<OrganizationInvitation>
+```
+
+## `CreateBulkOrganizationInvitationParams`
+
+<Properties>
```

Comment:
`CreateBulkOrganizationInvitationParams` is an `Array` type - how should it get represented with `Properties`?